mass_mailing_unique/models/mail_mass_mailing_contact.py

```diff
@@ -16,9 +16,13 @@ def _check_email_list_ids(self):
             others = lists.mapped('contact_ids') - contact
 
             contact_email = contact.email.strip().lower()
-            other_emails = [e.strip().lower() for e in others.mapped('email')]
+            other_emails = {e.strip().lower() for e in others.mapped('email')}
             if contact_email in other_emails:
-                raise ValidationError(_(
-                    "Cannot have the same email (%s) more"
-                    "than once in the same list." % contact_email
-                ))
+                in_list = others.filtered(lambda o: o.email == contact_email) \
+                    .list_ids.mapped('name')
+                lists = ''.join(map(lambda l: '- {}\n'.format(l), in_list))
+                msg = _('''Email (%(contact_email)s) already in mailing list(s):
+                    %(lists)s
+                    Please use a different email address or remove (%(contact_email)s) from the \
```

Comment:
This is failing flake8:
```
mass_mailing_unique/models/mail_mass_mailing_contact.py:26:89: E501 line too long (97 > 88 characters)
mass_mailing_unique/models/mail_mass_mailing_contact.py:28:89: E501 line too long (95 > 88 characters)
```

Reply:
Hmm I tried by using `git rebase -i` but I don't know if I did that right?  I always squashed with merges before...

Reply:
oh dear, what should I do now to fix?

Reply:
Rebase against the correct branch and with all the remotes properly updated.